recipes/client.rb

```diff
@@ -173,13 +173,15 @@
 zap_directory '/etc/sensu/conf.d/checks' do
   pattern '*.json'
   notifies :create, 'ruby_block[sensu_service_trigger]', :immediately
+  not_if { node[:platform_family].include?("windows") }
 end
 
 include_recipe 'sensu::client_service' unless node['recipes'].include?('monitor::master')
 
 directory '/var/cache/sensu' do
   owner 'sensu'
   group 'sensu'
-  mode 0o0755
+  mode 0755
```

Comment:
Is this change necessary to make it work on Windows? It should be equivalent and your notation breaks in rubocop. I could of course also exclude octal literatal from rubocop, but rubocop should run through without offenses, since we use it in CI.